$OpenBSD: patch-libsrc_odbcsdk_include_sqltypes_h,v 1.1.1.1 2013/04/22 17:32:31 zhuk Exp $
This hackish header tries to detect of wchar_t type is available
in a hacky way, so provide a hack for him to our hackiders too.
--- libsrc/odbcsdk/include/sqltypes.h.orig	Tue Apr 11 21:56:16 2006
+++ libsrc/odbcsdk/include/sqltypes.h	Fri Sep  9 00:19:03 2011
@@ -392,6 +392,7 @@ typedef unsigned short SQLWCHAR;
 #  if defined(__cplusplus)		|| \
       defined(_WCHAR_T)			|| \
       defined(_WCHAR_T_DEFINED)		|| \
+      defined(_WCHAR_T_DEFINED_)	|| \
       defined(_WCHAR_T_DECLARED)        || \
       defined(_BSD_WCHAR_T_DEFINED_)
 typedef wchar_t SQLWCHAR;
